<launch>
	<node name="config_throttle" pkg="throttle" type="config_throttle.py" output="screen" />
</launch>